src/mrt/python_poetry/mod.rs

```diff
@@ -0,0 +1,31 @@
+use std::{path::PathBuf, fs};
+
+use crate::{archetypes::Archetype, runners::NoopPackageScriptRunner, package::{NoopPackageInfoExtractor, PackageInfoExtractor}};
+
+pub struct PythonPoetryArchetype {}
+
+impl Archetype for PythonPoetryArchetype {
+    fn get_id(&self) -> &str {
+        "python/poetry"
+    }
+
+    fn matcher(&self, path: &PathBuf) -> bool {
+        let pyproject = path.join("pyproject.toml");
+
+        if pyproject.exists() {
+            let contents = fs::read_to_string(pyproject)
+                .expect("Should have been able to read the file");
```

Comment:
Would you think about displaying name of the file which fails.

Reply:
Yes, will add better parsting in next PRs, will actually parse toml file and read it the same way as poetry does. Thank you.